### ISSUE TITLE
How to set logic on what items inside mongo collection should be exported?

### ISSUE THREAD

**Issue by speciose** [10gen-labs/mongo-connector#304]
Can't find how to do that.
For example. I have "Posts" collection and each post could be published or unpublished. (I have "isPublished" field for that)
How can I prevent all posts with `isPublished: false` from being exported to elasticsearch?

(ES instance is open on client so I shouldn't collect any private data)

**Comment by llvtt** [comment]
Hi speciose,

Tailing changes based on a query isn't supported by mongo-connector. I'll
tag this issue as a feature request and modify the title as a
reminder/tracker for this feature.

**Comment by speciose** [comment]
Hi. It's me again. I'm just wondering.
Does it mean that this feature will come up this year? Or it's very hard to implement and it will take a long time to release?

**Comment by llvtt** [comment]
This feature is hard to implement, because an update operation might change whether a document matches the query. A "match" type query like `{"published": true}` would be easy enough, but the cases like `{"someArray": {"$gt": 5}}` get a lot harder. This would have to involve writing code to evaluate a query inside mongo-connector itself to determine whether a document is still part of a query set. Otherwise, a server-side change is required to make this feature easier to implement correctly.

Long story short: this feature is probably a long time away.
